test/generate-json.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs';
+import translate from '../lib/sparqlAlgebra';
+import Util from '../lib/util';
+import * as Path from 'path';
```

Comment:
```suggestion
import * as path from 'path';
```
I know you like using `Path`, but several other files already use `path` so let's use that one 😄

Reply:
Ah, now I remember why I always use `Path`: it clashes with the local variable name `path`, which tends to be used quite often.
But ok, I'll just rename that variable here in this case :-)